.github/workflows/main.yml

```diff
@@ -21,7 +21,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby-version: [3.2.2]
+        ruby-version: [3.2.2, 3.3]
```

Comment:
**question:** if we're adding ruby 3.3... We might want to add all rubies currently supported by rails?

Reply:
Good point, I can add all the Ruby versions that are not EOL.